Sources/Amplitude/Amplitude.m

```diff
@@ -588,16 +601,18 @@ - (void)logEvent:(NSString *)eventType withEventProperties:(NSDictionary *)event
             return;
         }
         if ([eventType isEqualToString:IDENTIFY_EVENT] || [eventType isEqualToString:GROUP_IDENTIFY_EVENT]) {
-            (void) [self.dbHelper addIdentify:jsonString];
+            [self->_identifyBuffer addObject:event];
+            [AMPStorage storeIdentify:jsonString];
```

Comment:
Will this improve performance? Have you verified? Writing to the store still happen for each event.